setup-scripts/setup-speech-synthesis.sh

```diff
@@ -13,16 +13,21 @@ sudo apt install -y festival speech-dispatcher speech-dispatcher-festival festvo
 wget http://www.speech.cs.cmu.edu/cmu_arctic/packed/cmu_us_slt_arctic-0.95-release.tar.bz2
 bunzip2 cmu_us_slt_arctic-0.95-release.tar.bz2
 tar xf cmu_us_slt_arctic-0.95-release.tar
-sudo mkdir -p /usr/share/festival/voices/english/
-sudo mv cmu_us_slt_arctic /usr/share/festival/voices/english/cmu_us_slt_arctic_clunits
 
+if [[ $(lsb_release -cs) == "bionic" ]]; then
+  sudo mkdir -p /usr/share/festival/voices/english/
+  sudo mv cmu_us_slt_arctic /usr/share/festival/voices/english/cmu_us_slt_arctic_clunits
+else
+  sudo mkdir -p /usr/share/festival/voices/us/
+	sudo mv cmu_us_slt_arctic /usr/share/festival/voices/us/cmu_us_slt_arctic_clunits
+fi
+  
 # done with downloaded files
 cd "$INSTALL_DIR"
 rm -rf /tmp/vx
 
 # set up festival voice
 sudo cp config/speechd.conf /etc/speech-dispatcher/
-sudo systemctl restart speech-dispatcher
```

Comment:
why are we removing this?

Reply:
in the newer versions of speech-dispatcher, it doesn't rely on a systemd unit. It gets launched by a different part of X, so the systemd unit doesn't work. Worse, it's always inactive and cannot be started, which throws an error if you try to restart it.